### PR TITLE
Samples upgraded to 1.2.0 and switched to programmatic config

### PR DESCRIPTION
**Pull request by sergeybykov**
On hold because samples are to be updated only after final 1.2.0 NuGets are published.

### CODE REVIEW

**Comment by sergeybykov** [comment]
Can be merged now that 1.2.0 NuGets are published.

File: Samples/GPSTracker/Host/OrleansHostWrapper.cs [hunk 1]
```diff
@@ -73,14 +75,13 @@ public bool Stop()
 
         private void Init()
         {
-            siloHost.LoadOrleansConfig();
+            //siloHost.LoadOrleansConfig();
```
Comment (jason-bragg) [inline Samples/GPSTracker/Host/OrleansHostWrapper.cs]: Why comment rather than remove?

File: Samples/GPSTracker/GPSTracker.Common/AzureHelper.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.WindowsAzure.ServiceRuntime;
+
+namespace GPSTracker.Common
+{
+    /// <summary>
+    /// Helper class to isolate from Microsoft.WindowsAzure.ServiceRuntime.dll and its failures to load outside of Azure
+    /// </summary>
+    public class AzureHelper
```
Comment (jason-bragg) [inline Samples/GPSTracker/GPSTracker.Common/AzureHelper.cs]: nit: 'Helper' is a poor naming convention that can become an unstructured catchall for junk code.  Can you be more explicit about what this is class is?  Maybe AzureDeployment?  DeploymentEnvironment? ..

File: Samples/HelloWorld/Tests/HelloWorldSiloTests.cs [hunk 3]
```diff
@@ -40,7 +40,7 @@ public static void ClassCleanup()
             // Optional. 
             // By default, the next test class which uses TestignSiloHost will
             // cause a fresh Orleans silo environment to be created.
-            StopAllSilos();
+            StopAllSilosIfRunning();
```
Comment (jason-bragg) [inline Samples/HelloWorld/Tests/HelloWorldSiloTests.cs]: Nit.  Odd naming.  Isn't it assumed we'd only stop running silos?  If we want to be explicit about them running in the call name, StopAllRunningSilos would seem less clumsy.

Comment (sergeybykov) [inline Samples/HelloWorld/Tests/HelloWorldSiloTests.cs]: `TestingSiloHost` is defined in `TestingSiloHost`. Can't rename it here.

**Pull request merged**